components/hab/src/cli.rs

```diff
@@ -75,6 +75,18 @@ pub fn get() -> App<'static, 'static> {
                     "Ring key name, which will encrypt communication messages")
             )
         )
+        (@subcommand bldr =>
```

Comment:
This is great, we might want to consider moving the `job` sub command under here at some point